src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -783,6 +783,204 @@ TIP: If you plan to use another DB version, it is important to get the correspon
+....
+# rm database
+\c rm
+alter table nodesourcedata alter column infrastructuretype TYPE oid USING infrastructuretype::oid;
+\q
+
+
+# scheduler database
+\c scheduler
+alter table environment_modifier_data alter column value TYPE oid USING value::oid;
+alter table job_data alter column input_space TYPE oid USING input_space::oid;
+alter table job_data alter column out_space TYPE oid USING out_space::oid;
+alter table job_data alter column global_space TYPE oid USING global_space::oid;
+alter table job_data alter column user_space TYPE oid USING user_space::oid;
+alter table job_data alter column description TYPE oid USING description::oid;
+alter table job_data_variable alter column variable_value TYPE oid USING variable_value::oid;
+alter table job_data_variable alter column variable_model TYPE oid USING variable_model::oid;
+alter table job_data_variable alter column variable_description TYPE oid USING  variable_description::oid;
+alter table script_data alter column script TYPE oid USING script::oid;
+alter table script_data alter column url TYPE oid USING url::oid;
+alter table selection_script_data alter column script TYPE oid USING script::oid;
+alter table selection_script_data alter column url TYPE oid USING url::oid;
+alter table task_data alter column description TYPE oid USING description::oid;
+alter table task_data alter column java_home TYPE oid USING java_home::oid;
+alter table task_data alter column working_dir TYPE oid USING working_dir::oid;
+alter table task_data_variable alter column variable_value TYPE oid USING variable_value::oid;
+alter table task_data_variable alter column variable_model TYPE oid USING variable_model::oid;
+alter table task_data_variable alter column variable_description TYPE oid USING variable_description::oid;
+alter table plannedworkflow alter column failure_message TYPE oid USING failure_message::oid;
+alter table rule_error alter column error_message TYPE oid USING error_message::oid;
+alter table submittedrule alter column failure_message TYPE oid USING failure_message::oid;
+\q
+
+# PCA database
+\c pca
+alter table variable alter column name TYPE oid USING name::oid;
+alter table variable alter column value TYPE oid USING value::oid;
+\q
+....
+In the second step, we remove large objects stored in these tables by executing the following SQL queries.
+....
+# rm database
+vacuumlo -v rm
+vacuumdb -f -v -d rm -t pg_largeobject
+
+# scheduler database
+vacuumlo -v scheduler
+vacuumdb -f -v -d scheduler -t pg_largeobject
+
+# pca database
+vacuumlo -v pca
+vacuumdb -f -v -d pca -t pg_largeobject
+
+# check autovacuum
+SHOW autovacuum;
+....
+
+**Notes:**
+
+ * The command vacuumdb is executed implicitly if AutoVacuum is enabled.
+ * The command vacuumlo consumes a lot of RAM and disk space depending on the number of large objects to be removed.
+
+
+===== Download the Postgres driver
+The Postgres driver have to be downloaded and added to the addons folder of ProActive (SCHEDULER_PATH/addons/)
```

Comment:
... driver has to be ... and added inside the addons ...

these 2 sections should be before "Prepare the database purge operation"